### ISSUE TITLE
CQL-to-ELM Translator - Signature annotations missing from included libraries

### ISSUE THREAD

**Issue by sanumula** [cqframework/clinical_quality_language#451]
When Signatures are enabled, only the referencing library when translated to
ELM has the signature annotations whereas the included libraries don't. I see that the translator is not passing the signature information while processing the included library at [1] and hence it is defaulting to SignatureLevel.None

[1] https://github.com/cqframework/clinical_quality_language/blob/master/Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/LibraryManager.java#L95

**Example:**
You can reproduce the issue by executing the translator on the top-level cql library i.e., ReferencingLib.cql shown below.

**RootLibrary.cql:**

```
library RootLibrary

define RootLibrarySum: Sum({1,2,3})
```

**BaseLib.cql:**

```
library BaseLib

include RootLibrary called Root

define BaseLibSum: Sum({10,20,30})
```

**ReferencingLib.cql:**

```
library ReferencingLib

include BaseLib called Base

define ReferencingLibSum: Sum({100,200,300})
```

The ELM generated from the above CQL is attached to this ticket.
[cql_elm_without_signature.txt](https://github.com/cqframework/clinical_quality_language/files/3593084/cql_elm_without_signature.txt)

Is this an issue with the translator or am I missing something?

**Comment by sanumula** [comment]
I submitted a [PR](https://github.com/cqframework/clinical_quality_language/pull/452) that addresses this issue.

**Comment by sanumula** [comment]
@brynrhodes Thanks for the review and merging the PR. Could you please let us know when will be the next CQFramework release?

**Comment by brynrhodes** [comment]
Addressed in 1.3.19-SNAPSHOT and 1.4.8-SNAPSHOT
